testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/testcontainers/PortBinder.java

```diff
@@ -33,12 +32,9 @@ public PortBinder(EnvironmentOptions environmentOptions)
 
     public void exposePort(DockerContainer container, int port)
     {
+        container.addExposedPort(port); // Still export port, at a random free number, as certain startup checks require this.
```

Comment:
I would not expose anything at random port. It does not sound useful but it could cause flaky tests.

Reply:
Please see a comment - this is intentional (and it was that way before)

Reply:
I do see flaky test failures from time to time that says that port is already bound. My guts says it could be related and since it is not useful I see no reason to keep it.

Reply:
Let's treat this out of scope for this PR

Reply:
@kokosing those failures occur not on product tests but on the smoke/distributed tests that are exposing ports locally so that tests can access those containers.